docs/pages/application-structure/env-vars-config.md

```diff
@@ -65,12 +65,6 @@ export default async function () {
       server: {
         httpServer: customServer, // A custom HTTP server of your own.
         adapter: "native", // Default. Can also be 'socket.io' or a reference to a custom adapter.
-        port: process.env.PORT,
```

Comment:
Why are you removing all these things? 🤔

Reply:
To promote using the spec file to specify the port. this seems like an alternative to the spec itself, which I don't like.